### PR TITLE
Added inserttag post | lang = en

### PR DESCRIPTION
**Pull request by fenepedia**
Added inserttag post

### CODE REVIEW

File: manual/en/04-managing-content/insert-tags.md [hunk 1]
```diff
@@ -460,6 +460,11 @@ the current date or including lightbox images.
       <code>xml:lang="fr"</code> attribute if the doctype is XHTML.</td>
 </tr>
 <tr>
+<td><code>{{post::*}}</code></td>
+  <td>This tag will will be replaced by a given post-variable.
```
Comment (lionel-m) [inline manual/en/04-managing-content/insert-tags.md]: will ~~will~~ be ...

And I propose : `POST variable` instead of `post-variable`

**Review by lionel-m — CHANGES_REQUESTED** [review]

File: manual/en/04-managing-content/insert-tags.md (hunk 1 above)
Comment (fenepedia) [reply manual/en/04-managing-content/insert-tags.md]: Gerne. Mein Englisch ist nicht das beste :-)

Comment (lionel-m) [reply manual/en/04-managing-content/insert-tags.md]: Could you add the french translation please. PS: It's not needed to create a PR for each languages but only one is enough.

```
<tr>
<td><code>{{post::*}}</code></td>
  <td>Cette balise sera remplacée par une variable POST.
  Elle peut être utilisée pour afficher les champs soumis par un formulaire.</td>
</tr>
```

**Comment by lionel-m** [comment]
Thanks for you contribution!

File: manual/en/04-managing-content/insert-tags.md (hunk 1 above)
Comment (fenepedia) [reply manual/en/04-managing-content/insert-tags.md]: Ok, but how can I add another file to this PR?

Comment (lionel-m) [reply manual/en/04-managing-content/insert-tags.md]: I see now that you edit a file directly from github... In this context, I don't know if we can do that. I'll check if it's possible, if not I will make a new PR for the French translation.

**Review by lionel-m — APPROVED** [review]

File: manual/en/04-managing-content/insert-tags.md (hunk 1 above)
Comment (lionel-m) [reply manual/en/04-managing-content/insert-tags.md]: I think you can't modify several files using this method but the flow described [here](https://guides.github.com/introduction/flow/) is the solution.

I will create a PR for the French translation.

Comment (fenepedia) [reply manual/en/04-managing-content/insert-tags.md]: Wait, I have created a patch

Comment (lionel-m) [reply manual/en/04-managing-content/insert-tags.md]: #423 Thanks!

**Pull request closed without merging**